funnel/views/project.py

```diff
@@ -278,11 +278,12 @@ def new_project(self) -> ReturnView:
             project = Project(created_by=current_auth.user, account=self.obj)
             form.populate_obj(project)
             project.make_name()
+            # trigger signal to generate thumbnail image
+            project_data_change.send(self.obj)
```

Comment:
`self.obj` here is account, not project. Send `project`, and after it is added to the db session, but before commit.